expertise/service/README.md

```diff
@@ -0,0 +1,145 @@
+# Running the Server
+The server is implemented in Flask and can be started from the command line:
+```
+python -m expertise.service --host localhost --port 5000
+```
+
+By default, the app will run on `http://localhost:5000`. The endpoint `/expertise/test` should show a simple page indicating that Flask is running.
+
+## Accessing the Endpoints
+Access to all endpoints requires authentication in the header of the request.
+
+**`POST /expertise`**
+
+**Request Body:**
+
+```
+{
+  config: JSON object containing options for both create_dataset and run_expertise
+}
+```
+
+**Returns:**
+
+`{ user_id: str, job_id: int }`
+
+---
+
+**`GET /jobs`**
+
+**Request Body:**
+
+`{}`
+
+**Returns:**
+
+```
+{
+  'dataset': [
+    {
+      job_name: str,
+      job_id: int,
+      status: str,
+    }, ...
+  ],
+  'expertise': [
+    {
+      job_name: str,
+      job_id: int,
+      status: str,
+    }, ...
+  ]
+}
+```
+
+---
+
+**`GET /results`**
+
+**Request Body:**
+
+`{ job_id: int, delete_on_get: boolean }`
```

Comment:
Can anyone retrieve the results of any job or is there a restriction?

Reply:
At the moment, you can only access information about jobs that you have personally submitted:
https://github.com/openreview/openreview-expertise/blob/a86bddea5e48d9efd20d00f8fe1a0f1f1be7e0f8/expertise/service/queue.py#L351-L353
Where the `id` is extracted from the token:
https://github.com/openreview/openreview-expertise/blob/a86bddea5e48d9efd20d00f8fe1a0f1f1be7e0f8/expertise/service/routes.py#L42-L44

Reply:
Is the profile_id the id of the job?

Reply:
or do you show all the jobs associated to that user?

Reply:
I think we should have a way to query a single job id, like GET /jobs?jobId=[]